docs/contributing/devguide.md

```diff
@@ -10,28 +10,6 @@ cd external-secrets
 
 _Note: many of the `make` commands use [yq](https://github.com/mikefarah/yq), version 4.2X.X or higher._
 
-If you want to run controller tests you also need to install kubebuilder's `envtest`.
-
-The recommended way to do so is to install [setup-envtest](https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest)
-
-Here is an example on how to set it up:
-
-```
-go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
-
-# list available versions
-setup-envtest list --os $(go env GOOS) --arch $(go env GOARCH)
-
-# To use a specific version
-setup-envtest use -p path 1.20.2
-
-#To set environment variables
-source <(setup-envtest use 1.20.2 -p env --os $(go env GOOS) --arch $(go env GOARCH))
-
-```
-
-for more information, please see [setup-envtest docs](https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest)
```

Comment:
Developers don't have to install the setup-envtest explicitly anymore

- https://github.com/external-secrets/external-secrets/pull/2527